deepchem/models/tensorgraph/tensor_graph.py

```diff
@@ -984,6 +984,16 @@ def get_layer_variables(self, layer):
       return tf.get_collection(
           tf.GraphKeys.TRAINABLE_VARIABLES, scope=layer.variable_scope)
 
+  def get_layer_weights(self, layer):
+
+    layer_variables = self.get_layer_variables(layer)
+    if tfe.in_eager_mode():
+      return layer_variables
```

Comment:
This should probably be `return [v.numpy() for v in layer_variables]`.  You want it to return the values of the variables, not the variable objects themselves.

Reply:
Ah yes!
Has been fixed now.